host/reset_workflow_test.go

```diff
@@ -404,7 +404,7 @@ func (s *IntegrationSuite) TestResetWorkflow_NoDecisionTaskCompleted() {
 	s.Logger.Info("PollAndProcessWorkflowTask", tag.Error(err))
 	s.NoError(err)
 
-	events = s.getHistory(s.domainName, &types.WorkflowExecution{
+	s.getHistory(s.domainName, &types.WorkflowExecution{
```

Comment:
With errcheck (or similar), I guess this is probably preferable over `_ = s.getHistory(...)`, isn't it... since that could be ignoring an error, but `s.getHistory(...)` cannot.  Interesting.